### PR TITLE
Include generated module and also non-python files

### PR DESCRIPTION
**Pull request by changhiskhan**
also add STRING as valid optionkey

### CODE REVIEW

File: python/rikai/spark/sql/generated/__init__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#  Copyright (c) 2021 Rikai Authors
```
Comment (eddyxu) [inline python/rikai/spark/sql/generated/__init__.py:1]: will this file be deleted after a new `make generated`?

File: src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4 [hunk 2]
```diff
@@ -84,6 +84,7 @@ option
 
 optionKey
     : identifier
+    | STRING
```
Comment (eddyxu) [inline src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4:87]: is this relevant?

Comment (changhiskhan) [reply src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4:87]: limits you to variable names only. e.g., you can't specify a spark config variable name as the option key (e.g., "rikai.model.config")

File: python/rikai/spark/sql/generated/__init__.py (hunk 1 above)
Comment (changhiskhan) [reply python/rikai/spark/sql/generated/__init__.py:1]: i added touch in the makefile

File: src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4 (hunk 2 above)
Comment (eddyxu) [reply src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4:87]: as key, it should be an quanlifiedname , as `OPTIONS (rikai.model.config = 123)` but not a STRING `OPTIONS ("rikai.model.config" = 123)`?

Comment (eddyxu) [reply src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4:87]: Filed #212

File: python/rikai/spark/sql/generated/__init__.py (hunk 1 above)
Comment (eddyxu) [reply python/rikai/spark/sql/generated/__init__.py:1]: nit, i am confused whether this file will be regererated via `make antrl`.
If yes, then the content in this file will be lost next time we run `make antlr`.
If no, we dont need to `touch` in Makefile?

**Review by eddyxu — APPROVED** [review]
LGTM.  Left a question , but it does not block merging this PR

File: python/rikai/spark/sql/generated/__init__.py (hunk 1 above)
Comment (changhiskhan) [reply python/rikai/spark/sql/generated/__init__.py:1]: Locally it depends on whether you deleted the whole directory contents or just removed one file. If you just change the g4 file without touching the generated directory make will do nothing (at least on Ubuntu). So we can't really guarantee one way or the other but this at least ensures we don't run into the same situation where those python files don't get added to the wheel. Touch will do nothing if the file already exists so it's not an issue either.

Comment (changhiskhan) [reply python/rikai/spark/sql/generated/__init__.py:1]: This file is not by default generated by the antlr command

Comment (changhiskhan) [reply python/rikai/spark/sql/generated/__init__.py:1]: Init will have no contents other than the license. It's just to make that it's a python module to setup

**Pull request merged**